docs/src/gs/create-workload-cluster.md

```diff
@@ -98,6 +98,54 @@ clusterctl generate cluster <cluster-name>\
 --from cluster-template-oraclelinux.yaml | kubectl apply -f -
 ```
 
+## Create a workload cluster in an alternative region
+
+CAPOCI provides a way to launch and manage your workload cluster in multiple
+regions. Choose the `cluster-template-alternative-region.yaml` template when
+creating your workload clusters from the [latest released artifacts][latest-release].
+Currently, the other templates do not support the ability to change the workload
+cluster region. 
+
+Each cluster can be further configured with the parameters
+defined in [Workload Cluster Parameters](#workload-cluster-parameters) and
+additionally with the parameter below.
+
+| Parameter             | Default Value                                        | Description                                                                                                                        |
+|-----------------------|------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------|
+| `OCI_WORKLOAD_REGION` | Configured as [OCI_REGION][configure-authentication] | The [OCI region](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm) in which to launch the workload cluster. |
+
+The following example configures the CAPOCI provider to authenticate in
+`us-phoenix-1` and launch a workload cluster in `us-sanjose-1`.
+
+> Note: Ensure the specified image is available in your chosen region or the launch will fail.
+
+[Configure authentication][configure-authentication] and Initialize management cluster:
+
+```bash
+...
+export OCI_REGION=us-phoenix-1
+export OCI_TENANCY_ID=<tenancy-id>
+export OCI_USER_ID=<user-id>
+...
+   
+clusterctl init --infrastructure oci
```

Comment:
This example is incomplete and has no context. If this is meant to be added/modified from another example, the preceeding text needs to be more explicit, ie. something like "To configure a different region for the workload cluster, extend the preceeding configuration example with the following additional configuration parameters". I'm not sure if the terms are correct there, but hopefully you get what I mean.